### ISSUE TITLE
Hello all

### ISSUE THREAD

**Issue by codywalker77** [dotnet/efcore#27014]
Hello all

EF Core 5.0.0-rc.2.20475.6

Having this model :

```
  public class Nation
    {
        public Guid ID { get; set; }
        public string Name { get; set; }
        public virtual ICollection<Person> Persons { get; set; }
    }

    public class Person
    {
        public Guid ID { get; set; }
        public string Name { get; set; }
        public virtual Nation Nation {get;set;}
    }

```

Is there a way to achive something like this?

```
public class MyDBContext : DbContext
    {
        public MyDBContext()
        {
            this.Database.EnsureCreated();
        }

        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        {
            base.OnConfiguring(optionsBuilder);
            optionsBuilder.UseSqlServer(@"XXX");
        }
        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            base.OnModelCreating(modelBuilder);
            modelBuilder.SharedTypeEntity<Person>("BadPerson").HasKey(itm => itm.ID);
            modelBuilder.SharedTypeEntity<Person>("GoodPerson").HasKey(itm => itm.ID);
            modelBuilder.SharedTypeEntity<Nation>("BadNation").HasKey(itm => itm.ID);
            modelBuilder.SharedTypeEntity<Nation>("BadNation").HasMany(itm => itm.Persons).WithOne(itm=>itm.Nation); // i'd like to refer to BadPerson Table 
            modelBuilder.SharedTypeEntity<Nation>("GoodNation").HasKey(itm => itm.ID);
            modelBuilder.SharedTypeEntity<Nation>("GoodNation").HasMany(itm => itm.Persons).WithOne(itm=>itm.Nation); // i'd like to refer to GoodPerson Table
        }
    }

```
Besides this case, is there a way I can found some documentation that explain more What are the limitations of shered Types?

Thanks

__Originally posted by @GLuca74 in https://github.com/dotnet/efcore/issues/23243__

**Comment by ajcvickers** [comment]
@codywalker77 You should not be using release candidates after the GA/RTM release is published. Also, at this point we are unlikely to backport any non-critical fix to 5.0, which [goes out of support in May](https://docs.microsoft.com/en-us/ef/core/what-is-new/). Please test on the latest 6.0.1 release, and if the issue still exists, then attach a small, runnable project or post a small, runnable code listing that reproduces what you are seeing so that we can investigate.

**Comment by ajcvickers** [comment]
**EF Team Triage:** Closing this issue as the requested additional details have not been provided and we have been unable to reproduce it.

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*
